docker/lookaside.go

```diff
@@ -178,8 +185,16 @@ func (config *registryConfiguration) signatureTopLevel(ref dockerReference, writ
 			return url
 		}
 	}
-	logrus.Debugf(" No signature storage configuration found for %s", ref.PolicyConfigurationIdentity())
-	return ""
+	if euid := rootless.GetRootlessEUID(); euid != 0 {
+		// returns unprivileged default directory if no sigstore specified in configuration file
+		defaultUserDockerDirPath := filepath.Join(homedir.Get(), defaultUserDockerDir)
+		u := url.URL{Scheme: "file", Path: defaultUserDockerDirPath}
+		logrus.Debugf(" No signature storage configuration found for %s, use default unprivileged directory %s", ref.PolicyConfigurationIdentity(), u.String())
+		return u.String()
+	}
+	u := url.URL{Scheme: "file", Path: defaultUserDockerDir}
+	logrus.Debugf(" No signature storage configuration found for %s, use default %s", ref.PolicyConfigurationIdentity(), u.String())
+	return u.String()
```

Comment:
After all this settles and tests work, the code and documentation that assumes `configuredSignatureStorageBase` can be `nil` should be removed/updated.